README.md

```diff
@@ -51,11 +66,12 @@ Tableau is introducing development support for Sandboxed Extensions with Tableau
 
     **npm run start-sandbox**
 
-2. Launch Tableau (Tableau 2019.3 and later) and try the sample Sandboxed Extension in a dashboard. You can find the `.trex` file and sample code in the `Samples\UINamespace-sandboxed` folder.
+2. Launch Tableau (Tableau 2019.3 and later) and try the sample Sandboxed dashboard extension in a dashboard. You can find the `.trex` file and sample code in the `Samples\UINamespace-sandboxed` folder.
```

Comment:
Would most likely want to change this folder path once I create the two Dashboard and Viz extension folders

Reply:
Done